content/news/2020-12-17-struggles.md

```diff
@@ -0,0 +1,123 @@
+title: About my Outreachy internship with Mixxx DJ: Struggles
+authors: Aanyu Deborah Oduman
+tags: personal, outreachy, internship
+date: 2020-12-06 18:10:00
+summary: Everybody struggles. This blog post is about a struggle that i went through in the past two weeks. I ran into this blocker while using git (as i normally do) and making some improvements on the Mixxx manual. I write about how i got stuck and the measures that i took to get unstuck.
+
+####  I thought i knew git until this day - struggles
+
+I hate feeling stuck.
+Staring at your screen blankly, at the verge of ripping your hair out because you have exhausted all the possible solutions and none of them has worked.
+
+![stressed out]({static}/images/news/ahmad-gunnaivi-OupUvbC_TEY-unsplash.jpg)
```

Comment:
Please use lowercase filenames. Can you remove the random characters and make the filename a little more descriptive here?